README.md

```diff
@@ -40,7 +40,7 @@ In your `package.json` add a new script:
 "scripts" :{
   ...
   "cy:run": "cypress run", // It can be any cypress command with any argument
-  "cy:parallel" : "cypress-parallel -s cy:run -t 2 -d '<your-cypress-specs-folder>' -a '\"<your-cypress-cmd-args>\"'"
```

Comment:
`-d` still totally works just the same, not broken with this change! `--spec` just makes more sense to recommend, because it can now be a dir or a file or a glob.